extensions/blocks/image-compare/edit.js

```diff
@@ -110,25 +106,31 @@ const Edit = ( { attributes, className, clientId, isSelected, setAttributes } )
 					<div className="image-compare__image-after">
 						<ImgUpload
 							image={ {
-								id: imageAfterId,
-								url: imageAfterUrl,
-								alt: imageAfterAlt,
+								id: imageAfter.id,
+								url: imageAfter.url,
+								alt: imageAfter.alt,
 							} }
 							placeHolderLabel={ __( 'Image after', 'jetpack' ) }
 							onChange={ img => {
 								if ( img.media_type === 'image' || img.type === 'image' ) {
+									const { src } = photonizedImgProps( img );
```

Comment:
Can we do the same for the image before?

Reply:
heh, missed one. After I changed one for testing, I decided to refactor the attributes and got focused on that part. Updated 👍